scss/_patterns_pagination.scss

```diff
@@ -0,0 +1,77 @@
+@import 'settings';
+
+@mixin vf-p-pagination {
+  .p-pagination {
+    display: flex;
+    width: 100%;
+
+    &__link {
+      display: block;
+      flex: 1;
+      margin-top: 0;
+      padding: $sp-medium;
+      position: relative;
+
+      &:first-child {
+        padding-left: $sp-x-large;
+
+        @media (max-width: $breakpoint-x-small) {
+          display: block;
+          max-width: $sp-x-small;
+        }
+      }
+
+      &:first-child .p-pagination__arrow {
```

Comment:
Can you nest this styling in the `&:first-child` above.

Reply:
That'll break the linter... we only allow 3 levels deep.

Reply:
Line 32 is nested.